packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.service.js

```diff
@@ -21,4 +21,21 @@ export default class {
       },
     );
   }
+
+  getDatastoreLocation(serviceName, datacenterId, filerId) {
+    if (!datacenterId) {
+      return this.OvhHttp.get(
```

Comment:
use $http instead

Reply:
Didn't know that, was keeping the same coding style as other files, switching to $http